src/xmlstoredp.bnd

```diff
@@ -0,0 +1,19 @@
+             strpgmexp  pgmlvl(*CURRENT) signature('XMLSTOREDP 19039')

```

Comment:
we should use `signature(*GEN)` here. These are store procedure targets and only designed to be called by the database. It doesn't care about signatures, only that the symbol is exported. This will also prevent users from attempting to link to it (though, I suspect nobody would).